setup.sh

```diff
@@ -0,0 +1,11 @@
+cd frontend
+npm install
+
+npm run dev
+
+npm run build
+
+cd ../backend
+virtualenv -p python3 venv
```

Comment:
Might as well ensure that we have at least Python 3.6? The new pyuavcan will require that. Or it should be specified in the readme.